scripts/qa_crowbarsetup.sh

```diff
@@ -1467,6 +1467,10 @@ function onadmin_activate_repositories
 
 function onadmin_bootstrapcrowbar
 {
+    test -n "$upgrade_cloudsource" && {
+        export cloudsource=$upgrade_cloudsource
+    }
```

Comment:
This will also be executed during installation.
I don't think that this is wanted.